Command/DropDatabaseDoctrineCommand.php

```diff
@@ -28,10 +28,10 @@ protected function configure()
         $this
             ->setName('doctrine:database:drop')
             ->setDescription('Drops the configured database')
-            ->addOption('shard', null, InputOption::VALUE_REQUIRED, 'The shard connection to use for this command')
-            ->addOption('connection', null, InputOption::VALUE_OPTIONAL, 'The connection to use for this command')
-            ->addOption('if-exists', null, InputOption::VALUE_NONE, 'Don\'t trigger an error, when the database doesn\'t exist')
-            ->addOption('force', null, InputOption::VALUE_NONE, 'Set this parameter to execute this action')
+            ->addOption('shard', 's', InputOption::VALUE_REQUIRED, 'The shard connection to use for this command')
+            ->addOption('connection', 'c', InputOption::VALUE_OPTIONAL, 'The connection to use for this command')
+            ->addOption('if-exists', 'e', InputOption::VALUE_NONE, 'Don\'t trigger an error, when the database doesn\'t exist')
```

Comment:
Conflicts with `--env|-e`

Reply:
Hum, maybe `i` ? 🤔

Reply:
I wouldn't even shortcut this specific option 😕

Reply:
Good point!

Reply:
Alias removed 🙂

Reply:
👍